plugins/modules/azure_rm_notificationhub.py

```diff
@@ -0,0 +1,410 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2021 Praveen Ghuge (@praveenghuge), Karl Dasan (@karldas30)
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
```

Comment:
from __future__ imports must occur at the beginning of the file

```suggestion

from __future__ import absolute_import, division, print_function
__metaclass__ = type
```